generators/Exercises/Generators/HighScores.cs

```diff
@@ -30,6 +30,14 @@ public void Latest_score_should_not_change_after_calling_personal_best()
     var sut = new HighScores(new List<int> { 20, 10, 30, 3, 2, 1 });
     Assert.Equal(30, sut.PersonalBest());
     Assert.Equal(1, sut.Latest());
+}
+
+[Fact(Skip = ""Remove this Skip property to run this test"")]
```

Comment:
If you run the test generator for this exercise using `pwsh .\generate-tests.ps1 -Exercise high-scores`, you'll find that there is no newline between the new method and the old additional method. The fix for this is relatively easy. Instead of using one big string for both additional methods, add them separately:

```suggestion
}");

testClass.AdditionalMethods.Add(@"[Fact(Skip = ""Remove this Skip property to run this test"")]
```